config/preview/params.yaml

```
@@ -15,5 +15,5 @@ footer_scripts:
   - adroll: false
 
 hotjar:
-  id: 1022108
+  id: 1021060
```

To commit before merging:

```suggestion
  id: 1021060
```